.github/actions/upload/action.yml

```diff
@@ -43,3 +43,11 @@ runs:
       working-directory: ${{ runner.temp }}/shadow_build_dir/${{ inputs.source }}
       run: aws s3 cp ${{ inputs.artifact_name }} s3://qgroundcontrol/latest/${{ inputs.artifact_name }} --acl public-read
       shell: bash
+
+    - name: Create Continuous Release
+      if: ${{ github.event_name != 'pull_request' && github.ref_name == 'master' && !github.event.pull_request.head.repo.fork }}
```

Comment:
Did you test `github.ref_name = 'master'`?  sometimes `ref_name` isn't what we want

Reply:
I would skip the fork conditional, I don't its needed

Reply:
Are you creating a release every time you push to master? Is this the desired behavior?

Reply:
Rather than every commit, should it be every night/week?

Reply:
Daily builds upload to S3 on every merge so seems like it should match that.

Reply:
Also seems like this is way ore important on automating Stables. This just kinda duplicates the S3 thing which is referenced by the docs. I guess some people go here to get releases though.